### PR TITLE
Add ps util exit process

### PR DESCRIPTION
**Pull request by dcristoloveanu**
Add ps util exit process (needed for terminating processes as part of some tests (offload) when we do not want to create extra polluting dumps)

### CODE REVIEW

File: src/ps_util.c [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // Copyright (C) Microsoft Corporation. All rights reserved.
 
 #include <stdlib.h>
+#include <stdbool.h>
```
Comment (mattdurak) [inline src/ps_util.c:4]: >stdbool [](http://example.com/codeflow?start=10&length=7)

not used

**Review by mattdurak — APPROVED** [review]
:shipit:

File: inc/c_util/ps_util.h [hunk 2]
```diff
@@ -3,13 +3,18 @@
 #ifndef PS_UTIL_H
 #define PS_UTIL_H
 
+#ifndef __cplusplus
+#include <stdbool.h>
```
[the remark this answers is not in the document]
Comment (dcristoloveanu) [reply inc/c_util/ps_util.h]: Removed

---
In reply to: [627773618](https://github.com/Azure/c-util/pull/117#discussion_r627773618) [](ancestors = 627773618)

File: src/ps_util.c (hunk 1 above)
Comment (dcristoloveanu) [reply src/ps_util.c]: Removed

---
In reply to: [627773732](https://github.com/Azure/c-util/pull/117#discussion_r627773732) [](ancestors = 627773732)

**Pull request merged**